src/components/Tool/FixEmailAddresses/EmailValidationForm.tsx

```diff
@@ -0,0 +1,97 @@
+import { Grid, IconButton, TextField } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Form, Formik } from 'formik';
+import { useTranslation } from 'react-i18next';
+import * as Yup from 'yup';
+import { AddIcon } from 'src/components/Contacts/ContactDetails/ContactDetailsTab/StyledComponents';
+import { RowWrapper } from './FixEmailAddressPerson';
+
+const ContactInputField = styled(TextField, {
+  shouldForwardProp: (prop) => prop !== 'destroyed',
+})(({ destroyed }: { destroyed: boolean }) => ({
+  textDecoration: destroyed ? 'line-through' : 'none',
+}));
+
+interface EmailValidationFormEmail {
+  email: string;
+  isPrimary: boolean;
+  updatedAt: string;
+  source: string;
+  personId: string;
+  isValid: boolean;
+}
+
+interface EmailValidationFormProps {
+  index: number;
+  personId: string;
+  handleAdd?: (personId: string, email: string) => void;
+}
+
+//TODO: Implement during MPDX-7946
+const onSubmit = (values, actions) => {
+  actions.resetForm();
+};
+
+const EmailValidationForm = ({ personId }: EmailValidationFormProps) => {
+  const { t } = useTranslation();
+
+  const initialEmail = {
+    email: '',
+    isPrimary: false,
+    updatedAt: '',
+    source: '',
+    personId,
+    isValid: false,
+  } as EmailValidationFormEmail;
+
+  const validationSchema = Yup.object({
+    email: Yup.string()
+      .email(t('Invalid Email Address Format'))
+      .required('Please enter a valid email address'),
+    isPrimary: Yup.bool().default(false),
+    updatedAt: Yup.string(),
+    source: Yup.string(),
+    personId: Yup.string(),
+    isValid: Yup.bool().default(false),
+  });
+
+  return (
+    <Formik
+      initialValues={initialEmail}
+      validationSchema={validationSchema}
+      onSubmit={onSubmit}
+    >
+      {({ values, handleChange, handleBlur, isValid, touched, errors }) => (
+        <Form>
+          <RowWrapper>
+            <Grid container>
+              <ContactInputField
+                destroyed={false}
+                label={t('New Email Address')}
+                type="email"
+                name="email"
+                value={values.email}
+                onChange={handleChange}
+                onBlur={handleBlur}
+                error={touched.email && Boolean(errors.email)}
+                helperText={
+                  touched.email && Boolean(errors.email) ? errors.email : ''
+                }
+              />
+              <IconButton
+                type="submit"
+                color="primary"
+                disabled={!isValid || values.email === ''}
```

Comment:
Making it appear to the user that a change is occuring by using `isSubmitting` from formik.
```suggestion
{({ values, handleChange, handleBlur, isValid, touched, errors, isSubmitting }) => (

  ....
                disabled={!isValid || values.email === ''}
                loading={isSubmitting}
```

Reply:
`loading` does not exist on buttons (nor on any of the other elements in this DOM tree). I can make it disabled while submitting if you want.